#goal definition
cobots_core/Move[] moves
bool wait_for_finish
---
#result definition
bool status
string message
---
#feedback
string message
